delivery.yaml

```
@@ -50,7 +50,7 @@ pipeline:
       fi
       export IMAGE MULTIARCH_IMAGE
 
-      make deps cicheck staticcheck gosec
+      make deps check-fmt vet staticcheck gosec shortcheck
```

Move validators before tests to fail early